issues: fix ai icon colors

JonasBa opened the pull request:
Fix icon colors in chonk

JonasBa committed:
issues: fix ai icon colors

roaga reviewed (commented):
What about `autofixRootCause.tsx` and `autofixTimeline.tsx`? Do those need updating too?

And does this change the look of anything if chonk is disabled?

JonasBa committed:
issues: update timeline

roaga reviewed (dismissed):
LGTM from the Autofix side at least

roaga reviewed (commented):
This seems to break the pink highlight on the root cause timeline
<img width="598" alt="Screenshot 2025-04-14 at 8 00 20 AM" src="https://github.com/user-attachments/assets/f4d864bb-108f-4237-b853-08e7d5432fc8" />

JonasBa commented:
@roaga let me take a look here. It might be that color being a string and not a key isn't reporting errors here. We usually use `<Icon color='pink400'/>` which performs the lookup on theme, but I think this component passes it via style, so it would break.

JonasBa committed:
issues: perform color lookup

JonasBa commented:
Fixed by [eed66be](https://github.com/getsentry/sentry/pull/89479/commits/eed66be54cd83dc36771f7e09d6d229d4f7d91ca)

JonasBa committed:
issues: remove unused activeColor prop

JonasBa committed:
issues: fix ai icon colors (#89487)

`ColorConfig` export pollutes the global namespace, picking the field
from `TimelineItemProps` indicates the config to Timeline item
relationship without polluting the namespace.
